services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -343,6 +349,103 @@ open class HankeServiceImpl(
 
     // --------------- Helpers for data transfer towards database ------------
 
+    private fun checkAndHandleDataProcessingRestrictions(
+            incomingHanke: Hanke,
```

Comment:
Method comments for purpose needed. Something about GDPR etc

Reply:
added